odoo/tools/mail.py

```diff
@@ -290,6 +292,22 @@ def is_html_empty(html_content):
     tag_re = re.compile(r'\<\s*\/?(?:p|div|span|br|b|i|font)(?:(?=\s+\w*)[^/>]*|\s*)/?\s*\>')
     return not bool(re.sub(tag_re, '', html_content).strip())
 
+def parse_inline_template(text):
```

Comment:
Would have a parse_inline_template_block method that hides complexity of doing it instruction by instruction and return a completely rendered block. A blackbox you can call and that either returns a rendered text, either crashes with some rendering / validation errors (as already proposed before I think).

Reply:
I don't get it. Could you explain ?